docs/products/kafka/howto/kafka-klaw.rst

```diff
@@ -0,0 +1,138 @@
+
+* `klaw/cluter-api/src/main/resources`
+* `klaw/core/src/main/resources`
+
+Configure authentication protocol
+""""""""""""""""""""""""""""""""""
+Klaw supports the following authentication protocols to connect to Apache Kafka® clusters:
+
+* ``PLAINTEXT``
+* ``SSL``
+* ``SASL PLAIN``, ``SASL SSL`` 
+* ``SASL SSL (GSSAPI / Kerberos)`` 
+* ``SASL_SSL (SCRAM SHA 256/512)``
+
+.. Note:: If you are using ``PLAINTEXT``, you do not need to perform any additional configuration. 
+
+
+Configuration for SSL protocol
+""""""""""""""""""""""""""""""""
+If you are using SSL as the authentication protocol to connect the Apache Kafka® cluster to Klaw, you need to perform the following steps: 
+
+Retrieve SSL certificate files
+''''''''''''''''''''''''''''''
+You need to retrieve the Aiven for Apache Kafka SSL certificate files. Aiven for Apache Kafka® by default enables TLS security. Download the certificates from the service overview page in the Aiven console or via the :ref:`dedicated page <avn_service_user_kafka_java_creds>`.
+
+Considering you have already configured the :doc:`Java SSL keystore and truststore files </docs/products/kafka/howto/keystore-truststore>`, move the keystore named ``client.keystore.p12`` and truststore named ``client.truststore.jks`` into a directory that can be easily accessed and configured with Klaw. 
+
+Configure SSL properties 
+'''''''''''''''''''''''''
+After retrieving the SSL certificate files and configuring the SSL keystore and truststore files, you need to configure these SSL values in the ``application.properties`` file.
+
+1. Get the **Cluster ID** by clicking the copy icon on the **Clusters** page in the **Klaw web interface**.  
+2. Next, open the application.properties file located in the **klaw/cluster-api/src/main/resources** directory. 
+3. Configure the SSL properties to connect to Apache Kafka® clusters by editing the following lines:
+
+   ::
+
+      klawssl.kafkassl.keystore.location=client.keystore.p12
+      klawssl.kafkassl.keystore.pwd=klaw1234
+      klawssl.kafkassl.key.pwd=klaw1234
+      klawssl.kafkassl.truststore.location=client.truststore.jks
+      klawssl.kafkassl.truststore.pwd=klaw1234
+      klawssl.kafkassl.keystore.type=pkcs12
+      klawssl.kafkassl.truststore.type=JKS
+
+   * For the lines starting with ``klawssl``, replace ``klawssl`` with the Klaw Cluster ID.
+   * Replace ``client.keystore.p12`` with the path for the keystore and ``klaw1234`` with the password configured for the keystore file. 
+   * Replace ``client.truststore.jks`` with the path for the truststore and ``klaw1234`` with the password configured for the truststore file. 
+   * Save the ``application.properties`` file.
+
+   The following is an example of an ``application.properties`` file configured with Klaw Cluster ID (demo_cluster), keystore, and truststore paths and passwords. 
```

Comment:
```suggestion
   The following is an example of an ``application.properties`` file configured with Klaw Cluster ID (``demo_cluster``), keystore, and truststore paths and passwords. 
```

Reply:
This change removes the need of defining ``demo_cluster`` in our dictionary

Reply:
Remove this.